maquetado/index.html

```diff
@@ -11,11 +11,68 @@
 </head>
 <body>
   <h2 class="title">Front-End Tools</h2>
-  <div class="technology">
-    <img class="logo" src="./assets/react_icon.svg">
-    <h4 class="technology-title">ReactJS</h4>
-    <h4 class="technology-release">Release:2013</h4>
-    <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+  <div class="container">
+    <div class="wrap">
+      <div class="technology">
+        <img class="logo" src="./assets/react_icon.svg">
```

Comment:
Remember to add `alt` attribute. I would also use a more specific class name for your tech images since you could have more than one logo in your page. Something like `technology-logo`